src/main/java/com/scottlogic/competitionEngine/application/user/UserServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package com.scottlogic.competitionEngine.application.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class UserServiceImpl{
+
+    @Autowired
+    public UserServiceImpl() {}
+
+    @CrossOrigin
+    @GetMapping("/users/authorise")
```

Comment:
I would maybe map everything in this to `/users` by having `@RequestMapping("/users")` at the top (just underneath `@RestControler` this will mean that `/user` is appended to the start of every one of your mappings.

Also for CrossOrigin I would specify `http://localhost:3000/` (at least for just now) better than letting anything through!